src/components/dashboard/SearchBar.js

```diff
@@ -64,15 +64,15 @@ export default function SearchBar({addStock}) {
             id="search-bar"
             size="small"
             className={classes}
-            options={stockData["data"]}
-            getOptionLabel={(option) => option.security + ' (' + option.symbol + ')'}
+            options={Object.entries(stockData["top100symbols"])}
+            getOptionLabel={(option) => option[1] + ' (' + option[0] + ')'}
```

Comment:
If you wanna wanna preserve the names, you can destructure in the parameter definition. Dunno if using string interpolation is more readable, though.
```suggestion
            getOptionLabel={([symbol, title]) => `${title} (${symbol})`}
```